utils/package.json

```diff
@@ -32,7 +32,7 @@
         "test": "node ./out/test/runTest.js"
     },
     "dependencies": {
-        "@microsoft/vscode-azureresources-api": "^2.0.2",
+        "@microsoft/vscode-azureresources-api": "^2.0.4",
```

Comment:
Ideally we'd be using `@microsoft/vscode-azureresources-api@2.0.5` but I don't actually know how Alex releases it. However, an `npm update` will pick it up since the versions are compatible, and ultimately it doesn't actually matter because this package is declaring `@azure/ms-rest-azure-env` as a peer dependency too, so it will transit to consumers of this package.